blocks/form/components/repeat/repeat.js

```diff
@@ -1,4 +1,23 @@
 import { getId } from '../../util.js';
+import { subscribe } from '../../rules/index.js';
+
+/* Radio buttons within the same instance should have the same name,
+but different instances should have different group names */
```

Comment:
Still documentation is missing for the functions

Reply:
Added